src/components/Dashboard/Modals/AboutModal/AboutModal.tsx

```diff
@@ -50,6 +52,21 @@ export const AboutModal = ({ opened, closeModal, newVersionAvailable }: AboutMod
   const informations = useInformationTableItems(newVersionAvailable);
   const { t } = useTranslation(['common', 'layout/modals/about']);
 
+  const keybinds = [
+    { key: 'Mod + J', shortcut: 'Toggle light/dark mode' },
+    { key: 'Mod + K', shortcut: 'Focus on search bar' },
+    { key: 'Mod + B', shortcut: 'Open docker widget' },
+    { key: 'Mod + E', shortcut: 'Toggle Edit mode' },
+  ];
+  const rows = keybinds.map((element) => (
```

Comment:
Nested components should be avoided, if they only display static data.
Your array above is static and I suggest placing this outside the main component.

Reply:
I don't think it changes anything, I just copied the example from Mantine's docs

Reply:
👍